modules/efk-logging-clo-status-comp.adoc

```diff
@@ -0,0 +1,129 @@
+// Module included in the following assemblies:
+//
+// * logging/efk-logging-cluster-status.adoc
+
+[id="efk-logging-clo-status-example-{context}"]
+= Viewing the status of cluster logging components
+
+You can view the status for a number of cluster logging components.
+
+.Prerequisites
+
+* Cluster logging and Elasticsearch must be installed.
+
+.Procedure
+
+. Change to the `openshift-logging` project.
++
+----
+$ oc project openshift-logging
+----
+
+. View the status of the cluster logging pod:
+
+.. Get the name of an cluster logging pod:
++
+----
+$ oc get pods --selector name=cluster-logging-operator -o name
```

Comment:
I'm not following why this is in here as part of checking the status messages of the other components. That information will be in the `status` section of `clusterlogging instance`. It will not be a part of the operator pod status

Reply:
@ewolinetz This topic, if I recall, was intended to parallel a similar topic on Elasticsearch that made it to 4.1 [1]. Check the instance, check the operator pod, deployment, replica set, etc, which I took from the test case. [2]
Is there any benefit to looking at the operator pod status? Such as to view the events?

[1] https://docs.openshift.com/container-platform/4.1/logging/efk-logging-elasticsearch-status.html
[2] https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-21450

Reply:
the only time you would do that is to determine why the operator couldn't be placed. That is independent of checking component statuses and not really a feature of the operator IMO

Reply:
@ewolinetz Should I be seeing the new status messages in the Cluster Logging instance at this time? Here is the Cluster Logging instance I see in my cluster, built from a recent nightly.
http://pastebin.test.redhat.com/781576

Reply:
@mburke5678 you would only see messages pulled from the scheduler if the pods were not available or able to be deployed. Since all your pods are `Ready` I would expect you to not see any conditions which is what your pastebin shows.

To test seeing components I would add a nodeSelector to a component which does not match any nodes, you should see a condition be updated and shown in the CR status after that.